.github/actions/build-deps/action.yml

```diff
@@ -48,9 +48,13 @@ inputs:
     default: c6f0b66318f8da6917fb4681103f7303b1836194
     type: string
   libzip:
-    description: Version of libzip to build
+    description: Version of libzip to build (Qt6 only)
```

Comment:
libzip works with Qt6 on Android?

Reply:
Not properly, but it works with Qt6 on desktop. KF6 KArchive will presumably work on Android though, but I don't think it's out yet still.